File: python/examples/example_vesselslist.py
from vesselfinder_api import VesselFinderApi
from vesselfinder_api.exceptions import ApiErrorException

v = VesselFinderApi(userkey='-- Input your userkey here --', errormode=False, save_last_info=True)

try:
    print(v.vessels_list())
    print(v.get_last_info())
except ApiErrorException as e:
    print(e)

File: php/examples/example_vesselslist.php
<?php
include __DIR__ . '/../VesselFinderApi.php';

use API\VesselFinderApi;

$client = new VesselFinderApi('-- Input your userkey here --');

try {
    $vessels_list = $client->vesselsList();
    print_r($vessels_list);
    print_r($client->getLastInfo());
} catch (\Exceptions\ApiErrorException $e) {
    print_r($e->getMessage() . PHP_EOL);
}